lib/sles4sap.pm

```diff
@@ -134,8 +134,17 @@ sub prepare_profile {
     }
 
     # Restart systemd-logind to ensure that all new connections will have the
-    # SAP tuning activated
-    systemctl 'restart systemd-logind.service';
+    # SAP tuning activated. Since saptune v2, the call to 'saptune solution apply'
+    # above can make the SUT change focus to the x11 console, which may not be ready
+    # for the systemctl command. If the systemctl command times out, change to
+    # root-console and try again. Run the first call to systemctl with
+    # ignore_failure => 1 to avoid stopping the test. Second call runs as usual
+    my $ret = systemctl('restart systemd-logind.service', ignore_failure => 1);
```

Comment:
I believe this systemctl call can be dropped because the saptune changes to the systemd-logind don't need a restart, and no user scenario contemplates this.  @angelabriel WDYT?

Reply:
Disagree. If you check the lines before the call to `systemctl`, you'll see that `saptune` is not always called. There are scenarios where `saptune` is not installed in the system and `sapconf` is used instead. Will wait for @angelabriel's input, but if this is not needed **with** saptune, I guess we can skip it only when `$has_saptune` is true. Nevertheless, I don't think it hurts to keep it.

Reply:
If saptune V2 is used to set (or unset) the 'UserTasksMax' limit, it will restart systemd-logind.service directly during the 'apply' or 'revert' operation. And saptune V1 does not set 'UserTasksMax' .
sapconf does the 'UserTasksMax'  setting in the postinstall of the package installation and needs a reboot to get the settings active. So may be in the case of sapconf installation without rebooting the system the restart of the systemd-logind.service may be helpful for the tests.
And yes, it does not hurt to keep it (with ignore_failure)